javascript/javascript3/week3/shopping.js

```diff
@@ -0,0 +1,128 @@
+// sometimes it showing 403 failed loading error
+const user1 = fetch(
+  "https://api.github.com/search/repositories?q=user:benna100"
+)
+  .then((response) => response.json())
+  .then((bennaData) => {
+    return `FullName : ${bennaData.items[1].full_name}<br>
+    url : ${bennaData.items[1].owner.url}<br>
+    Owner : ${bennaData.items[1].owner.login}`;
+  });
+
+const user2 = fetch(
+  "https://api.github.com/search/repositories?q=user:Mamidi001"
+)
+  .then((response) => response.json())
+  .then((mamidiData) => {
+    return `FullName : ${mamidiData.items[1].full_name}<br>
+    url : ${mamidiData.items[1].owner.url}<br>
+    Owner : ${mamidiData.items[1].owner.login}`;
+  });
+
+const user3 = fetch(
+  "https://api.github.com/search/repositories?q=user:amritpandey"
+)
+  .then((response) => response.json())
+  .then((amritData) => {
+    return `FullName : ${amritData.items[3].full_name}<br>
+    url : ${amritData.items[1].owner.url}<br>
+    Owner : ${amritData.items[1].owner.login}`;
+  });
+
```

Comment:
Having 3 snippets of code that does exactly the same is a sign that it should extracted into a function that performs the same, just accepting a parameter that deals with the dynamic part.